src/cpu/o3/O3CPU.py

```diff
@@ -52,3 +53,20 @@
         from m5.objects.X86CPU import X86O3CPU as O3CPU
 
     DerivO3CPU = O3CPU
+
```

Comment:
@powerjg I've added the citation now here and to the prefetcher. Is that correct?